app/mailers/shf_mailer.rb

```diff
@@ -1,2 +1,68 @@
-class ShfMailer < ApplicationMailer
+class ShfMailer < Devise::Mailer
+
+  # @see https://github.com/mailgun/mailgun-ruby/docs for examples
+  # @see https://github.com/plataformatec/devise/wiki/How-To:-Use-custom-mailer
+  #   for instructions on using this custom mailer with Devise
+
+  default from: ENV['SHF_NOREPLY_EMAIL']
+  layout 'mailer'
+
+  DEFAULT_MAILGUN_DOMAIN = 'sverigeshundforetagare.se'
```

Comment:
This sets up a default value for line 34.  However, if we have to change the default domain then we have to update it in two places.  My vote would be to just use the _single_ definition in `.env`.